### ISSUE TITLE
Management of data sent to the Pipeline Manager

### ISSUE THREAD

**Issue by servoz** [populse/populse_mia#98]
Currently, it is necessary to define the data used at the input of the Input_Filter brick, before dragging it into a pipeline editor (i.e. it is mandatory to use the "Send documents to the Pipeline Manager" button in the Data Browser before creating the Input_Filter brick).  

I think it's not desirable, because when developing pipelines, we may want to change the data at the input, without having to recreate everything (this is currently the case since we must have thought before the beginning of the pipeline creation, about the future data used at the input).  

It can also be quite annoying if we want to do an iteration (which uses the Input_Filter brick).  

Furthermore, it seems that if we forget to define in the Data Browser the data sent to the pipeline Manager, instead of having all the data, there is none in the input database_scans.  

On the other hand, if we have thought about defining data to be sent to the Pipeline Manager, after creating the Input_Filter brick if we return to the Data Browser and send new data to the Pipeline Manager, it is the data defined just before the Input_Filter brick was created that remain used.  

In short, it should be possible to be more flexible in the management of the data sent to the pipeline manager....
